js/sign.js

```diff
@@ -1,43 +1,40 @@
-// // 이메일 입력
-// const querySelector = (selector) => document.querySelector(selector);
-
-// const emailInput = querySelector("#email-input");
-// const emailErrorMessage = querySelector("#email-error-message");
-
-// emailInput.addEventListener("focusout", validateEmailInput);
-
-// function validateEmailInput(emailInput, emailErrorMessage, isValidateEmail, showEmailErrorMessage, hideEmailErrorMessage) {
-//   const email = emailInput.value.trim();
-
-//   if (email === "") {
-//     showEmailErrorMessage(emailErrorMessage, "이메일을 입력해 주세요.");
-//     return;
-//   }
-
-//   if (!isValidateEmail(email)) {
-//     showEmailErrorMessage(emailErrorMessage, "올바른 이메일 주소가 아닙니다.");
-//     return;
-//   }
-
-//   hideEmailErrorMessage(emailInput, emailErrorMessage);
-// }
-
-// function isValidateEmail(email) {
-//   return EMAIL_REGEX.test(email);
-// }
-
-// function showEmailErrorMessage(message) {
-//   emailErrorMessage.textContent = message;
-//   emailErrorMessage.classList.add("show");
-//   emailInput.classList.add("sign-input-error");
-//   emailInput.classList.add("error-focus");
-// }
-
-// function hideEmailErrorMessage() {
-//   emailErrorMessage.textContent = "";
-//   emailErrorMessage.classList.remove("show");
-//   emailInput.classList.remove("sign-input-error");
-//   emailInput.classList.remove("error-focus");
-// }
-
-// export { emailInput, emailErrorMessage, validateEmailInput, isValidateEmail, showEmailErrorMessage, hideEmailErrorMessage };
+// 이메일 유효성 검사
+export function isValidEmail(email) {
+  const EMAIL_REGEX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6}$/;
+  return EMAIL_REGEX.test(email);
+}
+
+// 비밀번호 유효성 검사
+export function isValidPassword(password) {
+  const PASSWORD_REGEX = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
+  return PASSWORD_REGEX.test(password);
+}
+
+// 오류 메시지 표시
+export function showErrorMessage(inputElement, messageElement, message) {
+  messageElement.textContent = message;
+  messageElement.classList.add("show");
+  inputElement.classList.add("sign-input-error");
+  inputElement.classList.add("error-focus");
+}
```

Comment:
함수 내부에서 사용하던 값들이 매개변수로 추출되면서, 함수명이 일반적으로 바뀐 것 까지 모듈화가 잘 된 것 같아요!
다만 매개변수가 많은 경우에 해당 함수를 사용하는 입장에서 인수의 순서를 정확히 지켜서 전달해야 하기 때문에 객체로 매개변수를 받는 것도 좋은 선택이 될 수 있을 것 같아요!
```suggestion
export function showErrorMessage({
  inputElement,
  messageElement,
  message
}) {
  messageElement.textContent = message;
  messageElement.classList.add("show");
  inputElement.classList.add("sign-input-error");
  inputElement.classList.add("error-focus");
}
```